app/helpers/handlers.js

```diff
@@ -40,26 +41,36 @@ const checkYarKeyReset = (thisAnswer, request) => {
   }
 }
 
-const insertYarValue = (field, request) => {
-  field = field.replace(SELECT_VARIABLE_TO_REPLACE, (_ignore, additionalYarKeyName) => (
-    field.includes('£') ? (formatUKCurrency(getYarValue(request, additionalYarKeyName) || 0)) : getYarValue(request, additionalYarKeyName)
-  ))
+const insertYarValue = (field, url, request) => {
+  field = field.replace(SELECT_VARIABLE_TO_REPLACE, (_ignore, additionalYarKeyName) => {
+    let result;
+    if (url === '1000-birds' && getYarValue(request, 'poultryType') === getQuestionAnswer('poultry-type','poultry-type-A1')) {
+        result = 'laying hens';
+    } else if (url === '1000-birds' && getYarValue(request, 'poultryType') === getQuestionAnswer('poultry-type','poultry-type-A2')) {
+        result = 'pullets';
```

Comment:
result should be returning the paragraph with the replaced content. Not just the word "pullets" or "hens"